101-signalr/azuredeploy.json

```diff
@@ -93,8 +101,13 @@
         "name": "[parameters('pricingTier')]"
       },
       "kind": "SignalR",
+      "identity": {
```

Comment:
how is this used?

Reply:
identity is assigned to a SignalR resource, then can be used to auth with SignalR Runtime(a.k.a SignalR data plane).  Details documented at: https://docs.microsoft.com/en-us/azure/azure-signalr/howto-use-managed-identity